Tasks/AzureAppServiceManageV0/azureappservicemanage.ts

```diff
@@ -86,6 +86,10 @@ async function run() {
             }
 
             tl.debug(`Resource Group: ${resourceGroupName}`);
+
+            let slotNameNotNeeded: boolean = (action == "Complete Swap" || action == "Swap Slots" || action == "Start Swap With Preview");
+            slotName = (!specifySlotFlag || slotNameNotNeeded) ? "production" : slotName;
```

Comment:
Can you make a contant string for defaultslotname with value production on the top.